openassessment/xblock/submission_mixin.py

```diff
@@ -438,6 +438,38 @@ def get_download_urls_from_submission(self, submission):
                 urls.append((file_download_url, '', ''))
         return urls
 
+    def get_files_info_from_user_state(self, username):
+        """
+        Returns the files information from the user state for a given username.
+
+        If the files information is present in the user state, return a list of following tuple:
+        (file_download_url, file_description, file_name)
+
+        Arguments:
+            username(str): user's name whose state is being check for files information.
+        Returns:
+            List of files information tuple, if present, else empty list.
+        """
+
+        files_info = []
+        user_state = self.get_user_state(username)
+        item_dict = self.get_student_item_dict_from_username(username)
+        if u'saved_files_descriptions' in user_state:
+            # pylint: disable=protected-access
```

Comment:
this will disable all errors after this line, so you don't have to disable it again on L#460

Reply:
ok